src/styles/block/reset.scss

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
it is forbidden to use `margin: 0`; in `*`. it's better to list them like 
`body,
h2,
p,
ul,
li,
button {
  margin: 0;
  padding: 0;
}`